### ISSUE TITLE
Docker standalone ksqldb-server  0.30.0 release 

### ISSUE THREAD

**Issue by xmlking** [confluentinc/ksql#10139]
Wonder when Docker standalone `ksqldb-server`  `0.30.0` version will be released?
last published version found is `0.29.0`
https://hub.docker.com/r/confluentinc/ksqldb-server

**Comment by swbradshaw** [comment]
I'm also wondering this as well. I would like to use the `run-ksql-test` which does not appear to work in the 0.29 release.

**Comment by raphaelauv** [comment]
![Screenshot from 2023-12-10 02-27-24](https://github.com/confluentinc/ksql/assets/10202690/321a2cb7-48c5-483d-97bf-0fce4b22bdb5)

**Comment by TheDarkFlame** [comment]
> ![Screenshot from 2023-12-10 02-27-24](https://private-user-images.githubusercontent.com/10202690/289592027-321a2cb7-48c5-483d-97bf-0fce4b22bdb5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI5NzM0MTIsIm5iZiI6MTcwMjk3MzExMiwicGF0aCI6Ii8xMDIwMjY5MC8yODk1OTIwMjctMzIxYTJjYjctNDhjNS00ODNkLTk3YmYtMGZjZTRiMjJiZGI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE5VDA4MDUxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiNzNmZjAyNTM3YzExNmU4ZjFkNjhkMWQ3NzMxZDY2MDBhZDU4MTEzMTE4YzAwZGI1OWQ2NzliMTc0MjQ4NjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3knZyZG1sJI8g7cZ5iBS-7MxaxJ5cbcunYUAfvofVS8)

I've pulled 7.5.2 (latest right now) and can confirm that the `run-ksql-test` script does not exist in this image. It looks like the only part of 0.30.0 that was pushed was the documentation.

**Comment by raphaelauv** [comment]
yeah this project is not well maintained by confluent ( they are mostly promoting flinkSQL , which makes sense )

My previous post was about showing how inconsistent is the documentation of confluent and this repo ( search for my previous issues  ) so no need to dislike it

**Comment by mootpointer** [comment]
I ran into the same issue, but it turns out that `run-ksql-test` is just a shell script. You can mount this script into the container and it will work! https://github.com/confluentinc/ksql/blob/891a6e65724a03f1eb59a2ad58037a6b42df7896/bin/run-ksql-test
